### DISCUSSION TITLE
Ability to appear in other apps fullscreen mode

### DISCUSSION THREAD

**Discussion by japanese-goblinn** [intellibar/main#16] [Ideas]
Hey folks! It would be nice if you could make IntelliBar appear when I'm in fullscreen mode, similar to how Alfred or Raycast do. I find this extremely convenient cause I work 98% of time in fullscreen

**Comment by astoilkov** [comment]
Yeah, we will do this for the next release.

**Reply by japanese-goblinn** [reply]
Great! Thank you for the amazing app
